.env

```diff
@@ -9,14 +9,14 @@ INTELOWL_TAG_VERSION=v1.6.1
 
 ###### Default (Production) ######
 
-COMPOSE_FILE=docker-compose.yml
+#COMPOSE_FILE=docker-compose.yml
```

Comment:
Please revert this change.

Reply:
I just change and commit update. Result is no change